app/src/main-process/squirrel-updater.ts

```diff
@@ -66,7 +66,13 @@ function getBinPath(): string {
 
 function resolveVersionedPath(binPath: string, relativePath: string): string {
   const appFolder = Path.resolve(process.execPath, '..')
-  return Path.relative(binPath, Path.join(appFolder, relativePath))
+
+  const computedPath = Path.relative(
+    binPath,
+    Path.join(appFolder, relativePath)
+  )
+
+  return Path.normalize(computedPath)
```

Comment:
It surprises me that we need to do a normalization step here, I thought `Path.relative` already took care of that, i.e:

```
> path.relative('c:\\one-win32-path', 'c:\\one-posix-path/foo/bar')
'..\\one-posix-path\\foo\\bar'
```

Unless you're sure this is necessary could we revert this change?

```suggestion
  return Path.relative(binPath, Path.join(appFolder, relativePath))
```